apps/src/gamelab/animationListModule.js

```diff
@@ -172,6 +172,16 @@ export function setInitialAnimationList(serializedAnimationList) {
     };
   }
 
+  // Convert frameRates to frameDelays.
+  for (let key in serializedAnimationList.propsByKey) {
+    let animation = serializedAnimationList.propsByKey[key];
+    if (animation.frameDelay === undefined && animation.frameRate !== undefined) {
+      animation.frameDelay = Math.round(30 / animation.frameRate);
+    } if (animation.frameDelay === null) {
```

Comment:
More generally, should we try to fill `frameDelay` if it is null _or_ undefined?  It seems like we want to make a strong assertion that every animation should have a nonzero frameDelay.  Also for `frameRate` it's good practice to check for what it is (number), not what it isn't (undefined). What if our check went more like this?

```javascript
if (!animation.frameDelay) {
  if (typeof animation.frameRate === 'number' && !isNaN(animation.frameRate)) {
    animation.frameDelay = Math.round(30 / animation.frameRate);
  } else {
    animation.frameDelay = 2;
  }
}
```